WordPress/Classes/Models/BloggingPromptSettingsReminderDays+CoreDataClass.swift

```diff
@@ -14,4 +14,21 @@ public class BloggingPromptSettingsReminderDays: NSManagedObject {
         self.sunday = remoteReminderDays.sunday
     }
 
+    func getActiveWeekdays() -> [BloggingRemindersScheduler.Weekday] {
+        return [
+            sunday,
+            monday,
+            tuesday,
+            wednesday,
+            thursday,
+            friday,
+            saturday
+        ].enumerated().flatMap { (index: Int, isReminderActive: Bool) in
+            guard isReminderActive else {
+                return nil
+            }
```

Comment:
Nitpick: maybe we could move this out of the `flatMap` closure and put it at the top? And return an empty array instead when the guard condition fails.